src/components/nodes-table/index.vue

```diff
@@ -24,7 +24,11 @@
       </v-row>
       <v-row class="ma-2" justify-start>
         <v-col cols="12">
-          <v-menu v-model="headersMenu" :close-on-content-click="false">
+          <v-menu
+            v-model="headersMenu"
+            :close-on-content-click="false"
```

Comment:
Is this necessary? I think it should work even without this line

Reply:
Yes, it's necessary.
Without this, the menu will close as soon as you click on a checkbox.